fieldservice_equipment_note/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# © 2023 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+{
+    "name": "Fieldservice Equipment Note",
+    "summary": """
+        Add new fields in the fsm order models.
```

Comment:
```suggestion
        Adds new model to have equipment observations in each fsm.order.
```
Also change readme/description
Generate Readme again with new changes